src/main/java/org/terasology/signalling/componentSystem/SignalSystem.java

```diff
@@ -349,6 +392,14 @@ private NetworkSignals getConsumerSignalInNetwork(Network2<SignalNetworkNode> ne
         return networkSignals;
     }
 
+    /**
+     * Checks for the most powerful signal on the network on a specific side and returns it
```

Comment:
Why "checks for ... and returns it" instead of "returns the most powerful ..." ?